Sources/UIScrollView+velocity.swift

```diff
@@ -62,8 +62,10 @@ extension UIScrollView {
     }
 
     func cancelDecelerationAnimations() {
-        layer.removeAnimation(forKey: "bounds")
-        horizontalScrollIndicator.layer.removeAnimation(forKey: "position")
-        verticalScrollIndicator.layer.removeAnimation(forKey: "position")
+        if !layer.animations.isEmpty {
```

Comment:
this removes a bunch of noise when debugging the animation code. This function gets called everytime exoplayer updates the sheet (every pixel). Removing animations, even when no animations exist, calls `layer.animations { didSet }` which does some computation. I think we can avoid this here. This might have a minor performance impact, but I found it better DX when debugging the animation code

Reply:
it should have a positive impact on performance, no?

Reply:
yep, but it should be very minor because nothing is happening really, just a few functions keep calling each other every pixel and then early return at the end because no animations exist at all. @rikner

Reply:
ok was just confused because you said "a minor performance impact, *but* ..." 😅

Reply:
got you 😄 , i meant it has some positive impact **but** i was mainly doing it for DX